INSTALLATION.md

```diff
@@ -62,7 +65,12 @@ cd talawa-admin
 
 ## Setting up NPM
 
-If you've followed the previous steps you should have already set up node.js on your system. [Click here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) for the official setup guide for npm.
+Best way to install and manage `node.js` is making use of node version managers. Two most popular node version managers right now are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). We'd recommend `fnm` because it's written in `rust` and is much faster than `nvm`. Install whichever one you want and follow their guide to set up `node.js` on your system.
+
+## Setting up Typescript
+
+As `talawa-admin` and `talawa-api` repositories are written using [Typescript](https://www.typescriptlang.org/), you will need to install typescript on your machine.
```

Comment:
1. `fnm` and `nvm` install node and the various modules locally and we recommend using them.
2. Add the command to install typescript. Use the API INSTALLATION.md as a guide

Reply:
@palisadoes 
`fnm` and `nvm` are version managers for `nodejs` which help to run multiple versions of `node` on a single machine whereas to install typescript we will have to use `npm` as given in the official typescript [docs](https://www.typescriptlang.org/download).
Refer : https://medium.com/@fs6158857/what-difference-between-npm-and-nvm-fa43e7631543#:~:text=In%20summary%2C%20NPM%20is%20a,play%20crucial%20roles%20in%20Node.
---
I will make the following changes in `Setting up typescript` section:
As `talawa-admin` and `talawa-api` repositories are written using `typescript`, you will need to install `typescript` on your machine. We recommend to install `typescript` globally using the following command
```
npm install typescript
```

You can also refer to the official [installation guide](https://www.typescriptlang.org/download)

Sir is this correct?

Reply:
OK